onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp

```diff
@@ -874,6 +875,7 @@ void RegisterDmlOperators(IMLOperatorRegistry* registry)
     }
 
     GpuDFTOperatorFactory::RegisterDFTKernel(registry);
+    DmlSTFTOperatorFactory::RegisterSTFTKernel(registry);
```

Comment:
>DmlSTFTOperatorFactory::RegisterSTFTKernel(registry); [](http://example.com/codeflow?start=4&length=53)

Yeah, once added to DirectML.dll, I'll be happy to see this temporary line deleted...